--- a/classes/presenter.php
+++ b/classes/presenter.php
@@ -36,7 +36,7 @@ abstract class Presenter
 	public static function forge($presenter, $method = 'view', $auto_filter = null, $view = null)
 	{
 		// determine the presenter namespace from the current request context
-		$namespace = \Request::active() ? ucfirst(\Request::active()->module) : '';
+		$namespace = \Request::active() ? ucfirst((string) \Request::active()->module) : '';
 
 		// create the list of possible class prefixes
 		$prefixes = array(static::$ns_prefix, $namespace.'\\');
